Makefile

```diff
@@ -442,7 +442,7 @@ qemu-version.h: FORCE
 			pkgvers="$(PKGVERSION)"; \
 		else \
 			if test -d .git; then \
-				pkgvers=$$(git describe --match 'v*' 2>/dev/null | tr -d '\n');\
+				pkgvers=$$(git rev-parse --short HEAD 2>/dev/null | tr -d '\n');\
```

Comment:
It switches the version string from `[release]-[tag]-[commit]` to just the commit hash.
I only managed to get `-dirty` before changing it as my local branch didn't have any tags. As far as I've noticed, nothing has broken because of that change.